README.md

```diff
@@ -66,11 +66,11 @@ You can make multiple calls in a query
 
 ```graphql
 query postAndTags {
-  post @rest(type: "Post", endpoint: "/post/1") {
+  post @rest(type: "Post", route: "/post/1") {
     id
     title
   }
-  tags @rest(type: "Tag", endpoint: "/tags") {
+  tags @rest(type: "Tag", route: "/tags") {
```

Comment:
I think this should maybe be `path`, not `route` - since that's the path part of a URL

Reply:
@stubailo yeah we didn't feel strongly either way. If you have an opinion towards path lets go with that!

Reply:
yeah it's not a strong opinion but route does feel more like an internal detail of the REST server

Reply:
I'm fine changing that too.